src/actions/code.js

```diff
@@ -12,6 +17,12 @@ import {
 import { JAVASCRIPT, PYTHON } from '../config/programmingLanguages';
 import { runPython } from '../runners/python';
 
+const setLanguage = data => dispatch =>
+  dispatch({
+    type: SET_LANGUAGE,
```

Comment:
I believe this is for programming language, so it should be called `SET_PROGRAMMING_LANGUAGE` and `setProgrammingLanguage` for consistency an clarity.

Reply:
DONE.